build/templates/onedark.template.vim

```diff
@@ -421,6 +421,21 @@ call s:h("xmlEndTag", { "fg": s:red })
 call s:h("xmlTag", { "fg": s:red })
 call s:h("xmlTagName", { "fg": s:red })
 
+" Perl
+call s:h("perlFiledescRead", { "fg": s:green })
+call s:h("perlMatchStartEnd",{ "fg": s:blue })
+call s:h("perlStatementFlow",{ "fg": s:red })
+call s:h("perlStatementStorage", { "fg": s:purple })
+call s:h("perlFunction", { "fg": s:purple })
+call s:h("perlMethod", { "fg": s:purple })
+call s:h("perlStatementFiledesc", { "fg": s:red })
+call s:h("perlVarPlain",{ "fg": s:blue })
+call s:h("perlSharpBang", { "fg": s:comment_grey })
+call s:h("perlStatementInclude", { "fg": s:purple })
+call s:h("perlStatementScalar",{ "fg": s:purple })
+call s:h("perlSubName",{ "fg": s:yellow })
+call s:h("perlSpecialString",{ "fg": s:cyan })
+
```

Comment:
You just need to [run the build step](https://github.com/joshdick/onedark.vim/blob/master/CONTRIBUTING.md#use-it) to create the real files from the templates and then this is good to go!